src/lib.rs

```diff
@@ -16,6 +17,77 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with Flue. If not, see <https://www.gnu.org/licenses/>.
 
+//! Flue is an efficient and secure actor runtime library.
+//!
+//! The fundamental building block in Flue is the **process**: a concurrent
```

Comment:
Processes do not exist in Flue. This is really unclear.
"the fundamental building block in Flue's philosophy and by extension intended use case is..."?